examples/chart/teleport-kube-agent/tests/deployment_test.yaml

```diff
@@ -22,8 +48,7 @@ tests:
       - equal:
           path: metadata.annotations.kubernetes\.io/deployment-different
           value: 3
-      - matchSnapshot:
-          path: spec.template.spec
```

Comment:
Something I didn't flag on the original PR; not sure why this is changed. It should still be `spec.template.spec` AFAIK.

Reply:
@webvictim Interesting, looks like helm tests actually fail when I rollback this change.

Reply:
I restored this change because it does seem like it's correct since we want to test against the entire resource spec, not just the template spec.

Reply:
I changed it because `spec.template.spec` is the `spec` of Pods, while deployment annotations stay in the root object. I didn't experience issues with tests even before the change, but I felt this was incorrect, and I fixed it. Sorry for not specifying it.